quickstats.py

```diff
@@ -10,23 +10,23 @@
 os.system('clear')
 
 # Character IDs:
-#Agnor - 354350
-#Ailith 298736
-#Arya - 354348
-#Ruby - #23559244
-#Sana #10788708
-#Typh - #607240
-#Lucky - 422436
-#Soraya - 1496718
-#Krenaxios - 46364292
-#Drusilia - 28192770
-#Akta - 58813
-#Komazur - 14820665
-#Zaag - 14820702
-#Qrix - 29117600
-#Cloron - 14820937
-
-campaignCharacter =  [14820937, 14820665]#354350, 298736, 354348, 23559244, 10788708, 607240]
+# Agnor - 354350
```

Comment:
IDE did not properly read the pound sign, so I re-entered it and added the space per good usage recommendation.